tracer/src/Datadog.Trace/Iast/StackWalker.cs

```diff
@@ -55,7 +55,13 @@ public static StackTrace GetStackTrace()
     public static bool TryGetFrame(StackTrace stackTrace, out StackFrame? targetFrame)
     {
         targetFrame = null;
-        foreach (var frame in stackTrace.GetFrames())
+        var frames = stackTrace.GetFrames();
+        if (frames is null)
```

Comment:
Maybe it's a good opportunity to unify behaviors between netcore and .net framework and also return false if frames is an empty array also

Reply:
Yep you're right, waiting for an answer on the comment above and i'll change it accordingly